docs/buildah-build.1.md

```diff
@@ -407,6 +407,14 @@ By default, Buildah manages _/etc/hosts_, adding the container's own IP address.
 
 Set the OS of the image to be built, and that of the base image to be pulled, if the build uses one, instead of using the current operating system of the host.
 
+**--output**, **-o**=""
+
+Output destination (format: type=local,dest=path)
+
+By default, a local container image is created from the build result. The --output (or -o) flag allows you to override this behavior, and a specify a custom exporter. For example, custom exporters allow you to export the build artifacts as files on the local filesystem instead of a Container image, which can be useful for generating local binaries, code generation etc.
+
+The value for --output is a CSV-formatted string defining the exporter type and options. Currently, local and tar exporters are supported. The local exporter writes the resulting build files to a directory on the client side. The tar exporter is similar but writes the files as a single tarball (.tar).
```

Comment:
```suggestion
The value for --output is a CSV-formatted string defining the exporter type and options. 

Valid _type_ values are:
- **local**: write the resulting build files to a directory on the client-side.
- **tar**: write the resulting files as a single tarball (.tar).
```

Reply:
An example or two would be helpful.  Looks like a good quick blog post once complete.

Reply:
do you handle a hyphen `-` with the tar option to write the output to stdout?  You might want to add that to the man page if so.

Reply:
Yes we handle `-` , added to docs with example.